scripts/core/filters/index.js

```diff
@@ -198,6 +198,11 @@ export default angular.module('superdesk.core.filters', [])
             _entry[propertyName].toLowerCase()
             : _entry.name.toLowerCase());
     })
+    .filter('sortByLabel', () => function(_collection, propertyName = 'label') {
```

Comment:
There is already `filter('sortByName')` and examples are:
 https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/vocabularies/services/VocabularyService.js#L54 or https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/templates/directives/TemplatesDirective.js#L20